grakn-graph/src/main/java/ai/grakn/graph/internal/RelationImpl.java

```diff
@@ -137,9 +138,8 @@ static String generateNewHash(RelationType relationType, Map<RoleType, Set<Insta
      */
     @Override
     public Relation addRolePlayer(RoleType roleType, Instance instance) {
-        if(roleType == null){
-            throw new IllegalArgumentException(ErrorMessage.ROLE_IS_NULL.getMessage(instance));
-        }
+        if(roleType == null) throw new IllegalArgumentException(ErrorMessage.ROLE_IS_NULL.getMessage(instance));
```

Comment:
There are some useful helper methods `Objects.requireNonNull` and Guava's `Preconditions.checkNotNull`. 

I'm not totally convinced you need a custom message for this either... after all, we would end up with a lot of messages if we did this for every API method!